server/mailet/rate-limiter-redis/src/main/scala/org/apache/james/rate/limiter/redis/RedisRateLimiter.scala

```diff
@@ -67,6 +67,10 @@ class RedisRateLimiterFactory @Inject()(redisConfiguration: RedisConfiguration)
       masterReplicaRedisConfiguration.redisURI.value.asJava,
       masterReplicaRedisConfiguration.readFrom)
 
+    case sentinelRedisConfiguration: SentinelRedisConfiguration =>
```

Comment:
Do you get `red` code in IDE?
![Screenshot_20240925_163820](https://github.com/user-attachments/assets/5813f6f6-6ab2-48e4-80f0-36d3344cb333)

Reply:
no